Cloud Application Manager/Getting Started/self-register-instances.md

```diff
@@ -0,0 +1,100 @@
+
+When it is enabled for a provider, a new button _Self-Register Scripts_ appears on the provider details page, allowing the user to retrieve the provider-specific run-once command snippet that is needed to bootstrap the virtual machine with in order to trigger the self-registration process for the instance.
+
+![Self-register enabled in an AWS provider](../../images/cloud-application-manager/getting-started/self-register_enabled.png)
+
+### Obtaining the self-registration script
+
+Once it has been enabled in the provider, when the user clicks the _Self-Register Scripts_ button, a dialog will show the command snippets to be used for both Linux and Windows based machines. It will also provide the ability to specify proxy settings, to allow instances being deployed behind a proxy server to contact back Cloud Application Manager through it for the self-registration process to be successful.
+
+![Self-register command snippets](../../images/cloud-application-manager/getting-started/self-register_scripts.png)
+
+The self-registration command snippets are customized per provider, and will only be valid for self-register instances being deployed on the corresponding underlying cloud account. You should copy the corresponding command snippet (either for Linux and Unix-like OS-es as Bash script, or for Windows as PowerShell script) and include it in the bootstrap process of the virtual machine definition or template to be made self-registered.
+
+When the self-register instances feature is enabled in a provider, the dropdown option changes to  _Disable Self-Register_, which you can use to disable the feature in the provider, so that no newly deployed instances will be self-registered, even if they include and execute the self-register snippet.
+
+### Managing self-registered instances
+
+A virtual machine deployed with self-registration will pop up in Cloud Application Manager as an imported instance automatically, belonging to the provider whose self-registration script was used.
+
+If the virtual machine belongs to a scaling group, a single instance will appear as imported in Cloud Application Manager, containing all the machines included in the scaling group.
+
+Once the instance is available in Cloud Application Manager, you can manage it as any other instance type, execute lifecycle events or edit it through the [Lifecycle Editor](../Core Concepts/lifecycle-editor.md), such as [deploying a Script Box](./deploy-script-box-to-registered-instance.md) on the instance.
+
+### Contacting Cloud Application Manager Support
+
+We’re sorry you’re having an issue in [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/). Please review the [troubleshooting tips](../Troubleshooting/troubleshooting-tips.md), or contact [Cloud Application Manager support](mailto:incident@CenturyLink.com) with details and screenshots where possible.
```

Comment:
I think this is the deprecated support text, perhaps you have used an old template.

Reply:
There are two flavors of the support section (one for ALM and another for CAM Platform, let's say). I tweaked this one (ALM) a little bit to talk about register an instance, but it should be it. Why do you think it is deprecated?